[2022-10-28 00:00:09,442] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: user_processing.process_user scheduled__2022-10-27T00:00:00+00:00 [queued]>
[2022-10-28 00:00:09,451] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: user_processing.process_user scheduled__2022-10-27T00:00:00+00:00 [queued]>
[2022-10-28 00:00:09,452] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-10-28 00:00:09,453] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-10-28 00:00:09,454] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-10-28 00:00:09,467] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): process_user> on 2022-10-27 00:00:00+00:00
[2022-10-28 00:00:09,476] {standard_task_runner.py:52} INFO - Started process 8342 to run task
[2022-10-28 00:00:09,479] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'process_user', 'scheduled__2022-10-27T00:00:00+00:00', '--job-id', '39', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpfjezlf4s', '--error-file', '/tmp/tmpj59ww0u9']
[2022-10-28 00:00:09,483] {standard_task_runner.py:80} INFO - Job 39: Subtask process_user
[2022-10-28 00:00:09,540] {task_command.py:369} INFO - Running <TaskInstance: user_processing.process_user scheduled__2022-10-27T00:00:00+00:00 [running]> on host 19f0eed441f9
[2022-10-28 00:00:09,590] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=process_user
AIRFLOW_CTX_EXECUTION_DATE=2022-10-27T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-27T00:00:00+00:00
[2022-10-28 00:00:09,617] {python.py:173} INFO - Done. Returned value was: None
[2022-10-28 00:00:09,628] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=process_user, execution_date=20221027T000000, start_date=20221028T000009, end_date=20221028T000009
[2022-10-28 00:00:09,656] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-10-28 00:00:09,679] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
